pt-BR/mail/chrome/communicator/wallet/wallet.properties

```diff
@@ -33,51 +33,51 @@
-SaveThisPasswordObscured= Use Password Manager to remember this password.
-SaveThisPasswordEncrypted = Use Password Manager to remember this password.
-xSaveTheseValuesObscured = Remember these values.  Stored values are not password protected.
-xSaveTheseValuesEncrypted = Remember these values.  Stored values are password protected.
-xSaveThisValueObscured = Remember this value.  Stored values are not password protected.
-xSaveThisValueEncrypted = Remember this value.  Stored values are password protected.
-UserName = User Name
-Password = Password
-SelectUserTitleLine = Select User
-NoPasswordsEverSaved = No passwords have ever been saved.
-Captured = Data has been captured.
-NotCaptured = There is nothing to capture.
-UnableToCapture = Unable to capture data.
-Converted = Stored data has been converted.
-AllDataWillBeCleared = All stored data will be erased.
-NotConverted = Unable to convert stored data.
-PasswordNotChanged = Unable to change the password that protects your sensitive information.
-PasswordExpired = Your sensitive information is now locked.
-PasswordNotExpired = Unable to lock your sensitive information.
-Caveat = Saving Passwords and Other Sensitive Information##Password Manager and Form Manager will save passwords, user names, and other sensitive information and enter them for you automatically when they are required.#This sensitive information is stored on your computer in a file that's difficult, but not impossible, to read.#If other people have access to your computer, you may want to password protect the stored sensitive information by choosing a Master Password.#If you choose to password protect your stored information, you will be asked to provide your Master Password from time to time.  This approach provides better security but is slightly less convenient.
-CaveatTitle = Alert
+newPassword = Escolha uma nova senha mestra.#(Deixe os campos em branco caso não deseje usar uma senha mestra.)#
+enterPassword = Digite uma senha mestra
+confirmPassword = Digite novamente a senha mestra
+confirmFailed_TryAgain? = As senhas não são idênticas. Por favor, digite a mesma senha mestra nos dois campos.
+SelectUser = Selecione um nome de usuário a ser inserido neste formulário.
+SelectUserWhosePasswordIsBeingChanged = Selecione o usuário cuja senha está sendo alterada.
+PasswordNotification = Para sua conveniência, o navegador pode memorizar suas informações de nome de usuário e senha para que você não precise digitá-las novamente no futuro. Antes de salvarmos as senhas no seu disco rígido, elas serão ocultas. Deseja habilitar esta funcionalidade?
+WalletNotification = Você pode salvar informações inseridas em formulários e, posteriormente, preenchê-las automaticamente em outros formulários. Para salvar essas informações, selecione Editar/Salvar Informações do Formulário no menu ao visualizar o formulário.
+noPrefills = Não existem campos que podem ser preenchidos automaticamente.
```

Comment:
CORRECTIONS NEEDED it should be noPrefills = Não existem campos que possam ser preenchidos automaticamente.